packages/dataviews/src/components/dataform/index.tsx

```diff
@@ -37,14 +38,18 @@ export default function DataForm< Item >( {
 		[ fields, form.visibleFields ]
 	);
 
-	return visibleFields.map( ( field ) => {
-		return (
-			<field.Edit
-				key={ field.id }
-				data={ data }
-				field={ field }
-				onChange={ onChange }
-			/>
-		);
-	} );
+	return (
+		<VStack spacing={ 4 }>
+			{ visibleFields.map( ( field ) => {
+				return (
+					<field.Edit
+						key={ field.id }
+						data={ data }
+						field={ field }
+						onChange={ onChange }
+					/>
+				);
+			} ) }
+		</VStack>
```

Comment:
## Testing instructions

In Gutenberg experiments (wp-admin/admin.php?page=gutenberg-experiments), enable the "Quick Edit in DataViews" experiment. In the Site Editor, go to Pages (`wp-admin/site-editor.php?postType=page`) and switch to Table view.

<img src="https://github.com/user-attachments/assets/e459a723-357a-4bf2-a678-8a4adfecf68d" alt="Layout dropdown" width="468">

Select a page row. Click the "Show quick edit sidebar" button.

<img src="https://github.com/user-attachments/assets/b22a4aa4-c961-4506-9294-521fa4e0834a" alt="Quick Edit button" width="320">

| Before | After |
|--------|-------|
|<img width="367" alt="Quick Edit, before" src="https://github.com/user-attachments/assets/025b40a8-e4ea-4f7c-8998-4d985f131bdb">|<img src="https://github.com/user-attachments/assets/b53450d6-2332-47bb-ae9e-0876dade95ac" alt="Quick Edit, after" width="371">|

---

These changes can also be seen in the Storybook for `DataForm`.